app/assets/stylesheets/components/firm/_all.scss

```diff
@@ -1,2 +1,3 @@
 @import 'settings';
 @import 'firm';
+@import 'firm_list';
```

Comment:
Fair point — didn't think about the directory structure beyond 'this is to do with firms'. But it's about the first BEM component right? Will action.